app/views/customers/_search_autocomplete.html.erb

```diff
@@ -3,11 +3,11 @@
     <li class="header"><b>People</b></li>
     <% if users.size <= limit %>
       <% users.each do |u| %>
-        <li class="clickable"><%= link_to u.name + "(#{u.customer.try(:name)})", "/users/edit/#{u.id}" %></li>
+        <li class="clickable"><%= link_to u.name + "(#{u.customer.try(:name)})", "/users/#{u.id}/edit" %></li>
```

Comment:
I've verified most of them. Not all of them. I'm not that familiar to the flow of the project yet. So I'm more relaying on the tests/specs and bit-part on my manual testing capabilities.